pkg/corerp/api/v20231001preview/testdata/environmentresource.json

```diff
@@ -45,6 +45,13 @@
           ]
         }
       },
+      "bicep": {
+        "authentication": {
```

Comment:
What if user provides nothing here? Like `"authentication": {}`?

Reply:
user could leave this empty, which mean no bicep config and we treat all bicep registries as public.